client/src/app/modules/events/event-card/event-card.component.scss

```diff
@@ -1,10 +1,20 @@
+@import "../../../../styles.scss";
+
 //card header
 //------------------------------------------
 
+.card {
+  background-color: $blue !important;
+  box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+
+  // hover out effect
+  transition: all 200ms ease-in-out;
+}
+
 .card:hover{
-  .card-header, .card-body {
-    color: #00a5db;
-  }
+  background-color: #00a5db !important;
```

Comment:
Here is a danger, that when theme changes, we'll not find this color changed as well

Reply:
What is your advice? Only to extract the color into variable and put it next to the others (in styles.scss)? Otherwise I don't know how to find the variable that affects all placeholder colors (if any variable exists)